core/utils.py

```diff
@@ -0,0 +1,49 @@
+import requests, re, jwt
+
+from django.core.exceptions import ValidationError
+from django.http            import JsonResponse
+from django.conf            import settings
+
+from users.models           import User
+
+class KakaoAPI:
+    def __init__(self, data):
+        self.client_id     = settings.KAKAO_CLIENT_ID
+        self.client_secret = settings.KAKAO_CLIENT_SECRET
+        self.redirect_uri  = settings.KAKAO_REDIRECT_URI
+        self.grant_type    = 'authorization_code'
+        self.user_uri      = 'https://kapi.kakao.com/v2/user/me'
```

Comment:
만약 다른 Kakao API를 사용하게 된다면 다른 uri를 또 클래스 attribute에 추가해줘야 합니다. 그럼 어차피 요청하는 uri가 반복된다면 해당 uri만 저장해놓고 메소드만 추가할 수 있게 하면 되지 않을까요?
```suggestion
        self.kakao_api      = 'https://kapi.kakao.com/v2'
        
    def ...(self):
        res = requests.get(
            f"{self.user_url}/...",
            ...
        )
        
    def ...(self):
        res = requests.get(
            f"{self.user_url}/...",
            ...
        )        
        
    def ...(self):
        res = requests.get(
            f"{self.user_url}/...",
            ...
        )
```